### ISSUE TITLE
Bigger text area for content

### ISSUE THREAD

**Issue by sophieschoice** [w3c/wai-wcag-em-report-tool#87]
When you reach the part in the tool where it is time to add found issues, I find the text area available a bit too small. I am typically a bit wordy explaining the issue and how to solve it. The small area does not make it easy to proofread/review a text that is a bit lengthy. Is it possible to make it bigger? Both in width and height?

**Comment by hidde** [comment]
Thanks Sophie for the feedback! 

The text area grows when you add content, so whenever you reach a new line, the area will become slightly bigger. If you see the 'Your report' box on the right hand side, you can close that too to get more space horizontally. 

You could also proofread on the View Report page or, if you want full control, by creating a HTML export and adding your own CSS to that.
